airflow-core/src/airflow/ui/public/i18n/locales/es/tasks.json

```
@@ -0,0 +1,10 @@
+{
+  "mapped": "Mapeado",
+  "notMapped": "No mapeado",
+  "retries": "Intentos",
```

```suggestion
  "retries": "Reintentos",
```

Try -> Intento
Retry -> Reintento